### PR TITLE
AUT-99 Revise README.md

### PR DESCRIPTION
**Pull request by dariacm**

### CODE REVIEW

**Review by drdaemos — APPROVED** [review]
💯 Great PR!

File: README.md [hunk 1]
```diff
@@ -73,93 +72,84 @@ are relevant for the technological stack of your organization, and replace `@lok
...
-JWT:
-eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJjbGFpbSI6InZhbHVlIiwiaWF0IjoxNjc2OTU4MDgwfQ.ssp6cX8Juv5a56VB-w4Nhdi1XmEyTsbPc9zBre2XylHnXvGdkx38GYYXAP9UDUAw4lkU7GE0FA8wlMitB3iFPKHLDWU8d-E1W0fV6GXAAngMMRrZeRCREYGx3FchEj8ufY4_7i4jGdA2ph8WOgrTqrpjJRjYYuClFYuCgH8QsRIeCnLo_UU3AhmqJ1FnUUXYK4lpk5ssvuUR3OZR4nexNZJEsIbL_584_FHc7duui7WMQ9fvBInNG4FRu3d2ZPC9RejAlV5sRk8H0HpLORM54h4SfeUXiapNnu7Td_in_3YmhdwBVugnxDdATUBMjHbSSjxn0UWCFD2whTxFOFS6rICGWhUMViRQ9fSljwiEaAkYGwRSOKB0McYXucYmxvjyBJo2ngFEGuymJi7Ow6cjNfri6BoiCoZwQfkvAzsrTYzR4lGV7lG7o4GpX4aoUTwnndvDtvxNzHtb4ssilfFFnvRQC63v9ybIRkIBhm9GaSepoPDw9lrblImnS13-WEPWy2l5_wIeYZUSPvlPIS3SV17b9ohGoNzk-axmB5QG1PvLYpZ2_t0z7h5od2vw5ZTPNOQ-RhNSu28REd4Mp0xHySYsn0ukf4kZHPUoGbMIuIMg6WhVTsz7V4n0nd1iPIjBfJjWM5dDSZfQvg4whwO1jeaE4BXxpjeiFqxf_tOT1QM
-Verified payload:
-{"claim":"value","iat":1676958080}
-```
+- Copy your public key to `JWT_PUBLIC_KEY` in your `.env` file as a one line string
```
Comment (fl-lokalise) [inline README.md:153]:
```suggestion
- Copy your public key to `JWT_PUBLIC_KEY` in your `.env` file as a one line string
- Restart the Node app to load the new public key 
```

Might not be the best phrasing, but will most likely save somebody

Comment (dariacm) [reply README.md:153]: Thank you @fl-lokalise. I will add this information.

Just to be aligned, you mean restarting the app with Docker? I think all I did was `source .env` to read the changes but maybe I recall wrongly

Comment (fl-lokalise) [reply README.md:153]: @dariacm No, my bad. I meant that the Node app that is started through `npm run start:dev` needs to be restarted as it reads and stores the public key in memory on start-up.

Comment (dariacm) [reply README.md:153]: Clear now, thanks! Good point, I added it 💯 https://github.com/lokalise/node-service-template/pull/184/commits/40d94e2ad94e2b4276246b061823c4ae25212853

Comment (fl-lokalise) [reply README.md:153]: @dariacm And thank you for taking in my suggestion and indulging my laziness to create another PR 🙇 
And big 👍 for improving the readme

**Pull request merged**